docker/mxnet-model-server

```diff
@@ -34,6 +33,16 @@ start_mms()
     models=""
     models_found=0
 
+    # Check if the process is already running , if so let the user know and exit
+    for pid in $(pgrep -f mxnet-model-server); do
+        if [ $pid != $$ ]; then
+            echo "[$(date)] : mxnet-model-server : Process is already running with PID $pid, run mxnet-model-server stop or mxnet-model-server restart"
```

Comment:
Good debug to have. Just finish up the restart command completely. I don't think restart can work without --mms-config option. Customers can just copy paste command from this.